### PR TITLE
Expose the `insertImageEditingWrapper` api 

### PR DESCRIPTION
**Pull request by juliaroldi**
Expose the `insertImageEditingWrapper` api to insert the resize and rotate wrapper into the selected image.

### CODE REVIEW

**Review by BryanValverdeU — APPROVED** [review]

**Comment by JiuqingSong** [comment]
Please wait

File: packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts [hunk 1]
```diff
@@ -349,6 +351,26 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
         );
     }
 
+    /**
+     * Insert the image editing wrapper in the selected image
+     */
+    public insertImageEditingWrapper() {
```
[the remark this answers is not in the document]
Comment (juliaroldi) [reply packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:357]: The startRotateAndResize only handles the DOM manipulation, this one also handles the changes in content model. 
This one gets the selected the image set isEditing in the image and when the image node is created call startRotateAndResize

File: packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts [hunk 2]
```diff
@@ -349,6 +351,26 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
...
+        if (this.editor) {
+            const selection = this.editor.getDOMSelection();
+            if (!this.editor.getEnvironment().isSafari) {
+                this.editor.focus(); // Safari will keep the selection when click crop, then the focus() call should not be called
+            }
+            if (this.editor && selection?.type == 'image') {
```
Comment (JiuqingSong) [inline packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:363]: no need to check this.editor again here

File: packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts [hunk 3]
```diff
@@ -349,6 +351,26 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
         );
     }
 
+    /**
+     * Insert the image editing wrapper in the selected image
+     */
+    public insertImageEditingWrapper() {
+        if (this.editor) {
+            const selection = this.editor.getDOMSelection();
+            if (!this.editor.getEnvironment().isSafari) {
```
Comment (JiuqingSong) [inline packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:360]: we should also only do focus if selection is image, right? so move this under line 363

File: packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (JiuqingSong) [reply packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:357]: Where is the code to set isEditing?

[the remark this answers is not in the document]
Comment (juliaroldi) [reply packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:357]: Under applyFormatWithContentModel function

**Pull request closed without merging**